config/prod/test-ubuntu.yaml

```diff
@@ -0,0 +1,21 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2.yaml
+stack_name: ubuntu-test
+parameters:
+  # The Sage deparment for this resource
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "Ubuntu base image"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.micro"
+  AMIId: "ami-010d7285c460bd379"
```

Comment:
Actually this is not going to work with managed-ec2-v2.yaml template because there's custom commands in that template to install stuff using the yum package manager.  https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/managed-ec2-v2.yaml#L500

I think to get this working we'll need to update the template to support distributions that use apt package manager.

Reply:
Are we going to work on allowing for ubuntu?  I want to push for the allowance of Amazon Linux and Ubuntu.  Theres not too much difference between linux and ubuntu, but I know personally I use Ubuntu over Linux. (honestly its just a difference between installation of dependencies)

Reply:
We will try.  I've already worked on it for a day but was not able to get it working.  The errors I ran into were are attached to https://sagebionetworks.jira.com/browse/IT-391